packages/office-ui-fabric-react/src/components/DatePicker/examples/DatePicker.Disabled.Example.tsx

```diff
@@ -0,0 +1,63 @@
+// @codepen
+import * as React from 'react';
+import { DatePicker, DayOfWeek, IDatePickerStrings } from 'office-ui-fabric-react/lib/DatePicker';
+import './DatePicker.Examples.scss';
+
+const DayPickerStrings: IDatePickerStrings = {
+  months: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+
+  shortMonths: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+
+  days: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'],
+
+  shortDays: ['S', 'M', 'T', 'W', 'T', 'F', 'S'],
+
+  goToToday: 'Go to today',
+  prevMonthAriaLabel: 'Go to previous month',
+  nextMonthAriaLabel: 'Go to next month',
+  prevYearAriaLabel: 'Go to previous year',
+  nextYearAriaLabel: 'Go to next year',
+  closeButtonAriaLabel: 'Close date picker'
+};
+
+export interface IDatePickerDisabledExampleState {
+  firstDayOfWeek?: DayOfWeek;
+}
+
+export class DatePickerDisabledExample extends React.Component<{}, IDatePickerDisabledExampleState> {
+  public constructor(props: {}) {
+    super(props);
+
+    this.state = {
+      firstDayOfWeek: DayOfWeek.Sunday
+    };
+  }
+
+  public render() {
+    const { firstDayOfWeek } = this.state;
+
+    return (
+      <div className="docs-DatePickerExample">
+        <DatePicker
+          firstDayOfWeek={firstDayOfWeek}
+          strings={DayPickerStrings}
+          placeholder="Select a date..."
+          ariaLabel="Select a date"
+          // tslint:disable:jsx-no-lambda
+          onAfterMenuDismiss={() => console.log('onAfterMenuDismiss called')}
+          // tslint:enable:jsx-no-lambda
```

Comment:
For simplicity and for what this example tries to demonstrate, I would say better to remove line 46, 47 and 48.

Reply:
@redoz to resolve the conflict pull the master and merge it into your branch and accept the incoming change form the master. Then run `npm run update-snapshots` again to ensure you are on the same page with the master.

Reply:
Hmmm... I see we have a screener story for the disabled TextField with a label on it but the screener did not block this PR for some reason... 🤔 @micahgodbolt any thoughts why the screener has a previously accepted screenshot for the disabled TextField with the new changes coming from this PR? For comparison take any other PR with the screener passing and look that the label for a disabled TextField is not grayed out. If this is happening on a regular basis lots of visual regressions can sneak in 😨

Reply:
@Vitalius1 generally I'd rebase instead of merging for a feature branch but if I understand you correctly you'd prefer the merge?

Reply:
I don't have a preference over rebase vs merge as long as it solves the issue with the conflicts 😄 . One thing for sure there might be some issues when trying mixin them both. As for why this happened with the screenshots conflicting... when someone adds a test in between your current master (from which you branched out) and the upstream master we have some `id` logic in the our tests that gets tripped like this. So if you rebase and update snapshots that should work too... (at least in theory 🤔 )